app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryControls.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import ExpandedStoryToolTip from './ExpandedStoryToolTip'
```

Comment:
I see that we're using semi-colons at the end of the imports. We should add it here too. :smile: